src/components/InvertedFlatList/BaseInvertedFlatList.js

```diff
@@ -137,12 +137,13 @@ class BaseInvertedFlatList extends Component {
                 // Web requires that items be measured or else crazy things happen when scrolling.
                 getItemLayout={this.props.shouldMeasureItems ? this.getItemLayout : undefined}
                 // We keep this property very low so that chat switching remains fast
-                maxToRenderPerBatch={1}
+                maxToRenderPerBatch={5}
                 windowSize={15}
-
                 // Commenting the line below as it breaks the unread indicator test
```

Comment:
We should remove these outdated comments

Reply:
👍🏻